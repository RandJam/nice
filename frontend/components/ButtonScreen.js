import React, {Component} from 'react';
import Button from './Button'
import {Platform, AppRegistry, Text, View, StyleSheet, TouchableHighlight, TouchableOpacity, TextInput } from 'react-native';

export default class ButtonScreen extends Component{
  render(){
    <View>
      <Button/>
    </View>
  }
}
