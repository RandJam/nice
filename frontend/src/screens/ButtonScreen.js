import React, {Component} from 'react';
import {Text, View } from 'react-native';
import Button from '../components/Button'

const util = require('util')

export default class ButtonScreen extends Component {
  render(){
      return (
      <View>
        <Text> ButtonScreen </Text>
        <Button />
      </View>
    )

  }
}
