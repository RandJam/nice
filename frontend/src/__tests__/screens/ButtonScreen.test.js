import React from "react";
import {shallow} from 'enzyme';
import ButtonScreen from "../../screens/ButtonScreen";
import renderer from "react-test-renderer";

describe ("ButtonScreen", () => {
  it("renders without crashing", () => {
    const rendered = renderer.create(<ButtonScreen />).toJSON();
    expect(rendered).toBeTruthy();
    expect(rendered).toMatchSnapshot();
  });

})
