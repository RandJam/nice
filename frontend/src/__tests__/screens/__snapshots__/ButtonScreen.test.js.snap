// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonScreen renders without crashing 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
     ButtonScreen 
  </Text>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Image
      source={1}
      style={
        Object {
          "height": 400,
          "width": 400,
        }
      }
    />
  </View>
</View>
`;
